config/jobs/kubernetes-csi/csi-driver-windows-poc/csi-driver-windows-poc-config.yaml

```diff
@@ -25,7 +25,7 @@ presubmits:
         - --check-leaked-resources
         - --cluster=
         - --gcp-zone=us-west1-b
-        - --extract=ci/latest
+        - --extract=ci/k8s-master
```

Comment:
is this the intended change? seems like the PR was a lot bigger before

how is this different than ci/latest?

Reply:
this PR is no longer needed. Thanks!